tests/admin-ui-tests/utils.ts

```diff
@@ -99,7 +88,7 @@ export const adminUITests = (
     let cleanupKeystoneProcess = () => {};
 
     afterAll(async () => {
-      await cleanupKeystoneProcess();
+      cleanupKeystoneProcess();
```

Comment:
why not `await`

Reply:
oh, it is a `let` that is changing to a `Promise<void>` below. I'll undo this